File: 06 - Exam Analysis/Vcl/other/w2w/1StClass4000/VCL6/cdemos/OutlookBar/infounit.cpp
//---------------------------------------------------------------------------
#include <vcl.h>
#pragma hdrstop

#include "infounit.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "fcDemoRichEdit"
#pragma resource "*.dfm"
TInfoForm *InfoForm;
//---------------------------------------------------------------------------
__fastcall TInfoForm::TInfoForm(TComponent* Owner)
        : TForm(Owner)
{
}
//---------------------------------------------------------------------------

File: 08 - CRM/Vcl2/1StClass4000/VCL6/demos/OutlookBar/InfoUnit.pas
unit InfoUnit;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  StdCtrls, ComCtrls, fcDemoRichEdit;

type
  TInfoForm = class(TForm)
    fcDemoRichEdit1: TfcDemoRichEdit;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  InfoForm: TInfoForm;

implementation

{$R *.DFM}

end.
